client/home/lib/virtualmachines/components/machineslist/machinedetails.coffee

```diff
@@ -217,16 +218,17 @@ generateSpecs = (machine) ->
 
   specs = []
 
+  providerName = jMachine.meta?.type
+  instanceType = jMachine.meta?.instance_type ? 't2.micro'
   size = jMachine.meta?.storage_size
 
+  { ram } = providerSpecs[providerName][instanceType]
+  { cpu } = providerSpecs[providerName][instanceType]
```

Comment:
this will fail if vm has different provider like `vagrant` there needs to be defaults for each spec.